Specifications/Language/3_Expressions/ValueLiterals.md

```diff
@@ -147,7 +147,7 @@ No literals exist for values of [operation type](https://github.com/microsoft/qs
 No literals exist for values of [function type](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language/4_TypeSystem#available-types); functions have to be [declared](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/1_ProgramStructure/3_CallableDeclarations.md#callable-declarations) on a global scope and new functions can be constructed locally using [partial application](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/3_Expressions/PartialApplication.md#partial-application).
 
 
-# Default Values
+## Default Values
```

Comment:
This puts "default values" on the same level as the different kinds of literals; we may need to demote L135, L141, and so forth to H3 lines to make this change work. On the other hand, default values may be going away soon enough anyway (#48)...

Reply:
I agree - this was a change to avoid a build warning in our repo. The Q#-lang files still need a structural edit pass (for example, this topic would benefit from some intro material and displaying the literal types in a table).

Reply:
The literal types are displayed in a table under the type system chapter. The introduction for this is the readme for the expressions. What exactly is generating a warning in the build? Is it enforcing that there is exactly one H1 header at the top of each file?

Reply:
Yes, only one H1 anywhere in the file.

Reply:
Ok, leaving the Default Values header here as H2 is fine - it will go away shortly in any case. But updating the ComparativeExpressions.md per my suggestion would be good.